frontends/web/src/components/aopp/vasp.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, JSX, RenderableProps } from 'preact';
+import * as styles from './vasp.css';
+import TwentyOneAnalyticsLogo from '../../assets/exchanges/logos/21analytics.png';
+import BitcoinSuisseLogo from '../../assets/exchanges/logos/bitcoin_suisse.png';
+import BittrLogo from '../../assets/exchanges/logos/bittr.png';
+import BityLogo from '../../assets/exchanges/logos/bity.png';
+import PocketBitcoinLogo from '../../assets/exchanges/logos/pocketbitcoin.svg';
+
+interface VASPProps {
+    fallback?: JSX.Element;
```

Comment:
i'd make this mandatory - you have a fallback to the fallback, which is going a bit too far imho :joy:

Reply:
only used once, all other occurences do not use it.

Reply:
my point is that maybe they should - having a fallback that has another fallback sounds weird and confusing. not a huge deal, just a nit.

Reply:
Yep, I don't like it to much too. Small explanation for the others reading this:

Usually we just display "domainname.tld" in light gray if the domain is not known.
But on the first AOPP view (user-approval) it should show a sentence:

if domain is known:

- logo
- domain
- generic sentence

if the domain is NOT known

- sentence including the domainname

Unfortunately we can't use the same translation for the generic sentence and the that contains the domain.